qnm_filter/gw_data.py

```diff
@@ -208,8 +208,10 @@ def condition(
 
         if flow or fhigh:
             cond_data = ss.filtfilt(b, a, raw_data)
+            cond_time = raw_time
         else:
             cond_data = raw_data
+            cond_time = raw_time
```

Comment:
You are defining `cond_time`  twice when `ds!=1` (see the below `if` statement). I think the easiest way is to move `cond_time = raw_time[::ds]` outside the  `if` statement, since it also works for `ds=1`.

Reply:
Ok, just changed that